slack.ts

```diff
@@ -0,0 +1,54 @@
+/** Error20:
+    - line 21: don't know how to declare call signature in tsc*/
+
+import * as querystring from "querystring";
+import fetch from "node-fetch";
+
+const tkn = process.env.SLACK_TOKEN;
+const domain = process.env.DOMAIN;
+
+async function sendMessage(message : string, config : string) : Promise<void> {
+    const msg = {
+        token: tkn,
+        channel: config,
+        as_user: false,
+        username: "HackGT",
+        text: message
+    };
+
+    const qs = querystring.stringify(msg);
+
+    const res = await
+        fetch(`https://${domain}.slack.com/api/chat.postMessage?${qs}`);
+    const json = await res.json();
+    console.log(json);
+}
+
+async function check(config : any): Promise<string> {
+    const res = await
+        fetch(`https://${domain}.slack.com/api/channels.list`);
+    const channels = await res.json();
+
+    const res2 = await
+        fetch(`https://${domain}.slack.com/api/groups.list`);
```

Comment:
Can you send a slack message to a `group` like this? How does that work?

Reply:
Groups are like private channels, I didn't know if we wanted to implement this as an option

Reply:
OK sounds good then.